acquia_lift.module

```diff
@@ -687,25 +640,6 @@ function acquia_lift_personalize_visitor_context() {
 }
 
 /**
- * Implements hook_personalize_agent_delete().
- */
-function acquia_lift_personalize_agent_delete($agent_data) {
-  if (!acquia_lift_is_testing_agent($agent_data)) {
-    return;
-  }
-  acquia_lift_agent_clear_verified_status($agent_data->machine_name);
```

Comment:
Same thing here... I don't see the verification status stuff being fully removed.